app/services/booking_bug/appointments.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+module BookingBug
+  class Appointments < Base
+    MAX_UPDATE_DAYS = 7 # used to limit number of entries queried to determine if an update ahs occurred.
+
+    # rubocop:disable MethodLength, AbcSize
```

Comment:
I feel like we should be listening to Rubocop here. There is a lot of behaviour in this method.